js/popup.js

```diff
@@ -58,40 +61,27 @@ function updateAllButtons() {
                 { id: 'guildCopyLink', data: guildCsv },
                 { id: 'guildExportCsvCopyLink', data: guildExportCsvString },
             ];
-
-            // add each character button to buttons
-            for (let i = 0; i < 9; i++) {
-                const charData = getCharacterCsv(cleanJson, i);
-                const characters = document.querySelectorAll('.characters > li > a');
-                buttons.push({ id: characters[i].id, data: charData })
-            }
```

Comment:
The character buttons don't work anymore. Adding this loop back fixes it

Reply:
Woops, added it back